tableau_utilities/scripts/merge_config.py

```diff
@@ -61,115 +50,36 @@ def merge_2_configs(existing_config, additional_config, debugging_logs=False):
     This assumes that the user is merging the same type of config
 
     Args:
-        existing_config: The current existing config
-        additional_config: The additional config to add
+        existing_config (dict): The current existing config
+        additional_config (dict): The additional config to add
+        debugging_logs (bool): True to print debugging logs
 
     """
-
     for column_name, column_details in additional_config.items():
-
-        if debugging_logs:
-            print(column_name)
-
         # If the column doesn't exist add it
         if column_name not in existing_config:
             if debugging_logs:
                 print("ADDING COLUMN", column_name, column_details)
             existing_config[column_name] = column_details
-
-        elif column_name in existing_config:
+        else:
             if debugging_logs:
                 print('ALTERING COLUMN:', column_name)
-                print('EXISTING COLUMN',  column_name, existing_config[column_name])
-                print('ADDITIONAL COLUMN', column_name, column_details)
-
-            # Set the values from the dictionaries
```

Comment:
@JustinGrilli  I don't understand why you are deleting all of this code and where the code is to replace it.

This code looks for each attribute in a column and takes the value from the new field if it's not none.   Where are you doing this now?

Reply:
Yeah so the code I replaced it with is doing the same thing, but without having to call out the old/new attribute explicitly. It does this by simply looping through the `column_details` from the `additional_config` below (starting on line 71). I'll add a comment below explaining a little more.